### PR TITLE
Release Notes 03/28/2021-04/03/2021

### PR DESCRIPTION
**Pull request by miwithro**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+* CSI Drivers will become default for [Kubernetes versions 1.21+](https://docs.microsoft.com/azure/aks/csi-storage-drivers).
+* Previous [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) deprecation was June 30th 2021. To better align with Kubernetes Upstream pod security policy (preview) deprecation will begin with Kubernetes version 1.21, with its removal in version 1.25. As Kubernetes Upstream approaches that milestone, the Kubernetes community will be working to document viable alternatives.
+* For all AKS clusters using Kubernetes v1.20+, CoreDNS will be upgraded to version 1.8.3. This will remove `resyncperiod` and `upstream`from the Kubernetes plugin.
+
+### Release Notes
+
+* Bug Fixes
+  * Fixed a bug in runc that caused pods to be stuck in containercreation in containerd 1.4.3 and 1.4.4.
```
Comment (weinong) [inline CHANGELOG.md:19]:
```suggestion
  * Fixed a bug in runc that caused pods to be stuck in container creation in containerd 1.4.3 and 1.4.4.
```

Comment (weinong) [reply CHANGELOG.md:19]: also, does it require node image upgrade from customers?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+  * Fixed a bug in VMAS that accidently enabled VMAS to be scaled down to 0.
+* Behavioral Changes
+  * Increased nslookup/nc timeout to 10s for Provisioning CSE in nodes.
+* Component Updates
+  * Removed Cross-namespace owner references in Azure Policy on AKS v1.20+.
+  * Added support for updating Windows password on Windows AKS nodes.
```
Comment (weinong) [inline CHANGELOG.md:25]: how?

Comment (miwithro) [reply CHANGELOG.md:25]: @keikhara did you post the steps anywhere?

File: CHANGELOG.md (hunk 1 above)
Comment (miwithro) [reply CHANGELOG.md:19]: @palma21 thoughts?

**Review by qpetraroia — APPROVED** [review]

**Review by miwithro — COMMENTED** [review]
Removed update.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+### Release Notes
+
+* Bug Fixes
+  * Fixed a bug in runc that caused pods to be stuck in container creation in containerd 1.4.3 and 1.4.4.
+  * Fixed a bug in VMAS that accidently enabled VMAS to be scaled down to 0.
+* Behavioral Changes
+  * Increased nslookup/nc timeout to 10s for Provisioning CSE in nodes.
+* Component Updates
+  * Removed Cross-namespace owner references in Azure Policy on AKS v1.20+.
+  * Updated omsagent to [March_release](https://github.com/microsoft/Docker-Provider/blob/ci_prod/ReleaseNotes.md#03262021--).
```
Comment (palma21) [inline CHANGELOG.md:25]: Usually we paste the version that can be checked on the image itself on the cluster.

Comment (palma21) [reply CHANGELOG.md:25]: In this case version: ciprod03262021

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,34 @@
...
+  * Updated Azure Confidential Compute Image to updated webhook and plugin version, to include a liveness probe.
```
Comment (palma21) [inline CHANGELOG.md:26]: To which version?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,34 @@
...
+* CSI Drivers will become default for [Kubernetes versions 1.21+](https://docs.microsoft.com/azure/aks/csi-storage-drivers).
+* Previous [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) deprecation was June 30th 2021. To better align with Kubernetes Upstream pod security policy (preview) deprecation will begin with Kubernetes version 1.21, with its removal in version 1.25. As Kubernetes Upstream approaches that milestone, the Kubernetes community will be working to document viable alternatives.
+* For all AKS clusters using Kubernetes v1.20+, CoreDNS will be upgraded to version 1.8.3. This will remove `resyncperiod` and `upstream`from the Kubernetes plugin.
```
Comment (palma21) [inline CHANGELOG.md:14]: *has been upgreaded?

**Review by palma21 — APPROVED** [review]

**Pull request merged**